src/brasil/gov/portlets/upgrades/v2000/configure.zcml

```diff
@@ -0,0 +1,24 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup">
+
+  <genericsetup:upgradeSteps
+      source="*"
+      destination="2000"
+      profile="brasil.gov.portlets:default">
+
+    <genericsetup:upgradeStep
+        title="Cook CSS resources"
+        description="There were changes in the CSS files, so we need to cook the resources."
+        handler="..cook_css_resources"
+        />
+
+    <genericsetup:upgradeStep
+        title="Cook JS resources"
+        description="There were changes in the JS files, so we need to cook the resources."
+        handler="..cook_javascript_resources"
+        />
```

Comment:
Why we need to cook JS and CSS if there are no changes in static resources here?

Reply:
in fact we don't need it; I just added it to have something to test.